docs/advanced/runtime/docs/index.md

```diff
@@ -24,6 +24,14 @@ This page currently contains links to the wiki-fied version of the runtime docum
 -   [Other topics](/docs/advanced/runtime/docs/other/)
 -   [Glossary](/docs/advanced/runtime/docs/glossary)
 
+### Runtime guidelines
+
+The following describe guidelines for specific areas of the runtime
+
+-   [Error handling/MonoError](/docs/advanced/runtime/docs/mono-error/)
+
+
+
```

Comment:
Nit: use only one newline here. I sometimes run the repo trough a markdown linter and it complains about that :)